README.md

```diff
@@ -1,59 +1,100 @@
-* [ReleaseNotes](https://wiki.freifunk.net/Berlin:Firmware/v1.0.2)
+More user-relevant information on the firmware can be found on the wiki page at: https://wiki.freifunk.net/Berlin:Firmware. There you can also find the
+* [Release Notes](https://wiki.freifunk.net/Berlin:Firmware/v1.0.2)
 * a tutorial ([en](https://wiki.freifunk.net/Berlin:Firmware:En:Howto) / [de](https://wiki.freifunk.net/Berlin:Firmware:Howto)) on router configuration
 
-For questions write a mail to <berlin@berlin.freifunk.net>, open a discussion with 
+If you have any questions, send an email to <berlin@berlin.freifunk.net>, open a discussion with 
 @freifunk-berlin/firmware or come to our weekly meetings.
-If you find bugs please report them at: https://github.com/freifunk-berlin/firmware/issues
+If you find bugs, please report them at: https://github.com/freifunk-berlin/firmware/issues
 
 ## Development
 
 ### Info
 
 For the Berlin Freifunk firmware we use vanilla OpenWrt with additional patches and packages.
-The Makefile automates firmware creation (applies patches, integrates custom packages and uses
-the Imagebuilder of OpenWrt).
+The Makefile automates the build of the firmware (applies patches, integrates custom packages and uses the
+the ImageBuilder of OpenWrt).
 
 The idea is to download OpenWrt via git, patch it with all patches in the `patches/` folder
-and configure it according to the configuration snippets in `configs/`. Then use the OpenWrt
-buildsystem to build all packages and the Imagebuilder. The packages to assemble into the final
-image for the router are taken from the imageflavors defined in `packagelist/`.
+patch it and configure it according to the configuration snippets in `configs/`. Then use the OpenWrt
+build system to build all packages and the image builder. The packages that will be assembled to the final image for the router
+image for the router are taken from the image flavors defined in `packagelist/`.
 
-### Build Prerequisites
+### Build prerequisites
 
 Please take a look at the [OpenWrt documentation](https://openwrt.org/docs/guide-developer/build-system/install-buildsystem?s[]=prerequisites#prerequisites)
-for a complete and uptodate list of packages for your operating system. 
+for a complete and up-to-date list of packages for your operating system. 
 
-On Ubuntu/Debian:
+Alpine:
 ```
-apt-get install git build-essential libncurses5-dev zlib1g-dev gawk time \
-  unzip libxml-perl flex wget gawk gettext quilt python libssl-dev
+apk add asciidoc bash bc binutils bzip2 cdrkit coreutils diffutils \
+findutils flex g++ gawk gcc gettext git grep intltool libxslt \
+linux-headers make ncurses-dev openssl-dev patch perl \
+python2-dev python3-dev rsync tar unzip util-linux wget zlib-dev
 ```
 
-On openSUSE:
+Arch / Manjaro:
+```
+# Essential prerequisites
+pacman -S --needed base-devel bash bzip2 git libelf libxslt ncurses \
+openssl python2 time unzip util-linux wget zlib
```

Comment:
according to @everloop2 (https://github.com/freifunk-berlin/firmware/pull/853#issuecomment-862465598) it should be "python".